package.json

```diff
@@ -21,7 +21,8 @@
     "benchmark": "turbo run benchmark --scope=astro",
     "lint": "eslint \"packages/**/*.ts\"",
     "format": "prettier -w .",
-    "version": "changeset version && pnpm install --no-frozen-lockfile && pnpm run format"
+    "version": "changeset version && pnpm install --no-frozen-lockfile && pnpm run format",
+    "test:temp": "cd packages/astro && pnpm run test:temp"
```

Comment:
nit: Not sure if we want to leave this in?

Reply:
Hah, thanks for catching that!  That's my super hacky "I need to run a test and test:match refused to play nice" trick